### PR TITLE
[Automation blogs] How to use playwright in cucumber

### PR DESCRIPTION
**Pull request by nploi**

### CODE REVIEW

File: content/posts/how-to-use-playwright-in-cucumberjs/example/features/search_job_openings_at_manabie.feature [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Feature: Search job openings at Manabie
+
+    Scenario: Bob search job openings at Manabie
+        Given Bob go to Manabie website
+        When Bob click Careers
+        Then See all job openings at Manabie
```
Comment (vctqs1) [inline content/posts/how-to-use-playwright-in-cucumberjs/example/features/search_job_openings_at_manabie.feature]:
```suggestion
    Scenario: Bob search job openings at Manabie
        Given Bob opens Manabie website
        When Bob goes to Careers section
        Then Bob sees all job openings at Manabie
```

Comment (vctqs1) [reply content/posts/how-to-use-playwright-in-cucumberjs/example/features/search_job_openings_at_manabie.feature]: I think go to better than click :)) more express meaning for non-tech person

File: content/posts/how-to-use-playwright-in-cucumberjs/index.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
++++
+date = ""
+author = "nploi"
+description = "How to use Playwright in cucumberjs"
+title = "How to use Playwright in cucumberjs"
+categories = ["e2e test"]
+tags = ["k8s", "playwright", "cucumberjs"]
```
Comment (vctqs1) [inline content/posts/how-to-use-playwright-in-cucumberjs/index.md]:
```suggestion
tags = ["e2e", "playwright", "cucumberjs"]
```

File: content/posts/how-to-use-playwright-in-cucumberjs/index.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
++++
+date = ""
+author = "nploi"
+description = "How to use Playwright in cucumberjs"
+title = "How to use Playwright in cucumberjs"
+categories = ["e2e test"]
```
Comment (vctqs1) [inline content/posts/how-to-use-playwright-in-cucumberjs/index.md]: Can we get same categories with thiiss post https://github.com/manabie-com/manabie-com.github.io/blob/main/content/posts/why-we-use-cucumber-for-end-to-end-testing/index.md

File: content/posts/how-to-use-playwright-in-cucumberjs/index.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+tags = ["k8s", "playwright", "cucumberjs"]
+slug = "how-to-use-playwright-in-cucumberjs"
++++
+
+This tutorial helps you run automate your test using Playwright in Cucumber.
+
```
Comment (vctqs1) [inline content/posts/how-to-use-playwright-in-cucumberjs/index.md]:
```suggestion
This tutorial helps you run `web` automation tests by using Playwright in Cucumber.
```

**Review by vctqs1 — APPROVED** [review]

**Review by phamnhuvu-dev — APPROVED** [review]

File: content/posts/how-to-use-playwright-in-cucumberjs/index.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
++++
+date = ""
+author = "nploi"
+description = "How to use Playwright in cucumberjs"
+title = "How to use Playwright in cucumberjs"
+categories = ["DevSecOps", "Testing"]
+tags = ["bdd", "end-to-end", "cucumber", "automation", "test"]
+slug = "how-to-use-playwright-in-cucumberjs"
++++
+
```
Comment (duytran1993) [inline content/posts/how-to-use-playwright-in-cucumberjs/index.md:10]: In our previous blog article [HERE](https://blog.manabie.io/2021/12/why-we-use-cucumber-for-end-to-end-testing/), we explained why we're using Cucumber for E2E test at Manabie, and provided a brief understanding about Cucumber.

This time, let's dive a little bit deeper!

File: content/posts/how-to-use-playwright-in-cucumberjs/index.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+This tutorial helps you run `web` automation tests by using Playwright in Cucumber.
```
Comment (duytran1993) [inline content/posts/how-to-use-playwright-in-cucumberjs/index.md]: This tutorial helps you run `web` automation tests, using Playwright in Cucumber.

File: content/posts/how-to-use-playwright-in-cucumberjs/index.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+
+Before begin, we will give a brief introduction to Cucumber and Playwright.
```
Comment (duytran1993) [inline content/posts/how-to-use-playwright-in-cucumberjs/index.md]: Before we start, let's cover a brief introduction to Cucumber again, and afterward, about Playwright.

File: content/posts/how-to-use-playwright-in-cucumberjs/index.md [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+    - After run:
+
+        ```bash
+        1 scenario (1 passed)
+        3 steps (3 passed)
+        0m03.739s (executing steps: 0m03.729s)
+        ```
+
+### Conclusion
+Cucumber and Playwright is great framework. I hope this article will be of some use to you. [Here is source code](/content/posts/how-to-use-playwright-in-cucumberjs/example), thank you.
```
Comment (duytran1993) [inline content/posts/how-to-use-playwright-in-cucumberjs/index.md]: Cucumber and Playwright are great frameworks. I hope this article will be of some use to you. [Here is source code](/content/posts/how-to-use-playwright-in-cucumberjs/example), thank you.

Curious for more about how we're tackling testing problems at Manabie? Join us as an Engineer **[HERE](https://manabie.breezy.hr/)**, or follow our blog for future articles.

**Review by vctqs1 — APPROVED** [review]

**Review by phamnhuvu-dev — APPROVED** [review]

**Review by nvcnvn — APPROVED** [review]

**Pull request merged**